doc/advanced.rst

```diff
@@ -371,9 +371,12 @@ The ``node`` sub-node will contain an expression of ``my_value``. Node-based
 tests also have access to the ``arguments`` node. This node will contain the
 various other arguments that have been provided to your test.
 
+.. versionadded:: 1.36
+    Dynamic tests support was added in Twig 1.36.
```

Comment:
it should say `1.36 and 2.6`, as we maintain 2 branches in parallel with minor versions in both.

Reply:
btw, this also impact the way we produce our changelog, as we would have to make it clear that this feature is also new in 2.5 (compared to 2.5)